gapis/api/templates/specific_gfx_api.cpp.tmpl

```diff
@@ -266,6 +272,27 @@
 {{end}}
 
 
+{{/*
+-------------------------------------------------------------------------------
+  Emits a *reduced* validity check for an indirected function pointer.
+-------------------------------------------------------------------------------
+*/}}
+{{define "FastGetIndirectedFunction"}}
+  {{AssertType $.Element  "string"}}
+  {{$annotation := index $.Annotations 0}}
+  {{if eq (len $.Annotations) 1}}
+    {{$.Element}} != 0
```

Comment:
Could `0` be replaced by `VK_NULL_HANDLE` here?

Reply:
No, this code knows nothing about Vulkan.

Reply:
Is it really worth special-casing 0 here?
Typically this will be a small number of calls, (vkGet*procaddr) and are not going to be in the performance-critical part of the code. This seems like added complexity.

Reply:
Hrm, or nevermind, this is recursive. Base-case.